### PR TITLE
Support first & last publication date.

### PR DESCRIPTION
**Pull request by benembery**

### CODE REVIEW

File: src/prismicio.AspNetCore/Document.cs [hunk 1]
```diff
@@ -12,10 +13,12 @@ public class Document : WithFragments
         public string Href { get; }
         public ISet<string> Tags { get; }
         public IList<string> Slugs { get; }
-        public string Slug => Slugs.Count > 0 ? Slugs[0] : "-";
+        public string Slug => Slugs.Count >  0 ? Slugs[0] : "-";
         public string Type { get; }
+        public DateTime? FirstPublicationDate { get; }
+		public DateTime? LastPublicationDate { get; }
```
Comment (Raph22) [inline src/prismicio.AspNetCore/Document.cs]: Fix indentation

File: src/prismicio.AspNetCore/Document.cs [hunk 2]
```diff
@@ -12,10 +13,12 @@ public class Document : WithFragments
         public string Href { get; }
         public ISet<string> Tags { get; }
         public IList<string> Slugs { get; }
-        public string Slug => Slugs.Count > 0 ? Slugs[0] : "-";
+        public string Slug => Slugs.Count >  0 ? Slugs[0] : "-";
```
Comment (Raph22) [inline src/prismicio.AspNetCore/Document.cs]: Delete the space

File: src/prismicio.AspNetCore/Document.cs [hunk 3]
```diff
@@ -79,12 +84,14 @@ public static Document Parse(JToken json)
             var uid = (string)json["uid"];
             var href = (string)json["href"];
             var type = (string)json["type"];
+            var firstPublicationDate = (DateTime?) json["first_publication_date"];
+			var lastPublicationDate = (DateTime?)json["last_publication_date"];
```
Comment (Raph22) [inline src/prismicio.AspNetCore/Document.cs]: Fix indentation

File: tests/prismicio.AspNetCore.Tests/DocTest.cs [hunk 4]
```diff
@@ -6,139 +6,151 @@
...
             Assert.Equal(48.877108, place.Latitude);
             Assert.Equal(2.333879, place.Longitude);
-		}
-	}
+        }
+
+        [Fact]
+        public void PublishDateTest()
+        {
+            var document = Fixtures.GetDocument("language.json");
```
Comment (Raph22) [inline tests/prismicio.AspNetCore.Tests/DocTest.cs]: language.json ==> seem to not exist. Are you sure this is what you wanted to write ?

Comment (benembery) [reply tests/prismicio.AspNetCore.Tests/DocTest.cs]: I'll complete #1 and merge into this branch which will resolve this, issue

**Pull request merged**